ci/scripts/r_deps.sh

```diff
@@ -25,6 +25,12 @@ source_dir=${1}/r
 pushd ${source_dir}
 
 # Install R package dependencies
+# install.packages() emits warnings if packages fail to install,
+# but we want to error/fail the build.
+# options(warn=2) turns warnings into errors
+${R_BIN} -e "options(warn=2); install.packages('remotes'); remotes::install_cran(c('glue', 'rcmdcheck', 'sys')); remotes::install_deps()"
+# Separately install the optional/test dependencies but don't error on them,
+# they're not available everywhere and that's ok
 ${R_BIN} -e "install.packages('remotes'); remotes::install_cran(c('glue', 'rcmdcheck', 'sys'))"
```

Comment:
I might be missing somethigngsubtle, but do we need line 34 here still? Isn't like 31 doing this (and then a bit more?)

Reply:
You're right, sloppy merge conflict resolution, I'll fix